client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -297,9 +298,36 @@ const Editor = (props: EditorProps): JSX.Element => {
     // TODO this should *probably* be set on focus
     editorRef.current = editor;
     data.editor = editor;
+
+    const storedAccessibilityModes = () => {
+      const accessibilityMode = {
+        isAccessibilityModeOn: props.inAccessibilityMode
+      };
```

Comment:
```suggestion
      const accessibilityMode = props.inAccessibilityMode;
```
It seems simpler just to use a boolean here.  The rest of the code naturally becomes cleaner if you do.

Reply:
Not sure about this, this is only to store the key in the local storage, for example: when a new user starts for the first time, or when local storage gets deleted.